frontend/src/components/NewAgreement/AgreementCreatedSuccessfully.tsx

```diff
@@ -1,41 +1,84 @@
 import { StringBank } from 'strings';
 import { useNavigate } from 'react-router-dom';
 import { GroupContext } from 'contexts/group';
+import handLogo from 'assets/icons/hand-v@3x.png';
+import styled from 'styled-components';
+import { ReactComponent as CopyIcon } from 'assets/icons/copy.svg';
 
-const AgreementCreatedSuccessfully: FC = () => {
+const CopyIconWrapper = styled(SvgIcon)`
+  svg path {
+    fill: #adb2b8;
+  }
+`;
+
+const CopyEL: FC<{ text: string }> = ({ text }) => {
+  return (
+    <IconButton
+      sx={{ marginLeft: '2rem', marginRight: '1rem' }}
+      onClick={() => {
+        navigator.clipboard.writeText(text);
+      }}
+    >
+      <CopyIconWrapper>
+        <CopyIcon />
+      </CopyIconWrapper>
+    </IconButton>
+  );
+};
+
+const AgreementCreatedSuccessfully: FC<{ agreementId: string }> = ({ agreementId }) => {
   const { t } = useTranslation();
   const { slug: groupSlug } = useContext(GroupContext);
   const navigate = useNavigate();
+  const AgreementText = `${window.location.origin}/${groupSlug}/agreement/${agreementId}`;
   return (
     <Container maxWidth="xs" sx={{ height: '100%' }}>
       <Stack alignItems="center" justifyContent="center" paddingX={6} spacing={2} height="100%">
-        <DoneRoundedIcon
-          fontSize="large"
-          sx={{ bgcolor: 'success.main', borderRadius: '50%', padding: 0.5 }}
-        />
+        <img width="165px" height="180px" src={handLogo}></img>
         <Typography variant="h2" textAlign="center">
           {t(StringBank.AGREEMENT_PUBLISHED_SUCCESSFULLY)}
         </Typography>
-        <Box />
-        <Button variant="contained" size="large" color="primary" fullWidth>
+        <Button variant="contained" size="large" sx={{ backgroundColor: '#8d54ea' }} fullWidth>
           {t(StringBank.VIEW_AGREEMENT)}
         </Button>
         <Button
           variant="contained"
           size="large"
           fullWidth
+          sx={{
+            marginBottom: '0.5rem !important',
+          }}
           onClick={() => navigate(`/${groupSlug}/active-agreements`)}
         >
           {t(StringBank.RETURN_TO_ALL_AGREEMENTS)}
         </Button>
-        <Divider flexItem sx={{ paddingY: 1 }} />
+        <Divider flexItem sx={{ marginTop: '2rem' }} />
         <Typography>{t(StringBank.SHARE_AGREEMENT)}</Typography>
-        <Stack direction="row" spacing={2} alignItems="center" justifyContent="center">
-          <Typography>stub for now</Typography>
+        <Stack
+          direction="row"
+          spacing={2}
+          alignItems="center"
+          justifyContent="center"
+          bgcolor="rgba(0,0,0,0.08)"
+          borderRadius="4px"
+          border="solid 1px #565d68"
+          padding="0 1rem"
+        >
+          <Typography
+            whiteSpace="nowrap"
+            textOverflow="ellipsis"
+            overflow="hidden"
+            fontSize="0.87rem"
+            fontWeight="normal"
+            color="#adb2b8"
```

Comment:
Also let's use text.primary etc. for color, instead of color code which is hard coded

Reply:
fixed